src/Adapter/Order/OrderProductQuantityUpdater.php

```diff
@@ -183,13 +193,23 @@ private function updateProductQuantity(
 
             $this->updateCustomizationOnProductDelete($order, $orderDetail, $oldQuantity);
         } else {
-            // Update product and customization in the cart
+            /**
+             * Update product and customization in the cart, last argument skip quantity check
```

Comment:
Same complex explanation (it's complex because it explains a complex issue), I suggest same change